lib/web/device_trust.go

```diff
@@ -44,6 +47,8 @@ func (h *Handler) deviceWebConfirm(w http.ResponseWriter, r *http.Request, _ htt
 	confirmToken := &devicepb.DeviceConfirmationToken{}
 	confirmToken.Id = query.Get("id")
 	confirmToken.Token = query.Get("token")
+	redirectURI := query.Get("redirect_uri")
```

Comment:
I think it's way to easy to mix up `redirectURI` (which could potentially lead to an external site) with `redirectTo` (which is safe).

Thoughts on renaming it to something like this?

```suggestion
	unsafeRedirectURI := query.Get("redirect_uri")
```